torch/_higher_order_ops/templated_attention.py

```diff
@@ -38,6 +39,41 @@ def __call__(
 templated_attention.__module__ = "torch.ops.higher_order"
 
 
+class TemplatedAttentionBackwardHOP(HigherOrderOperator):
+    def __init__(self):
+        super().__init__("templated_attention_backward")
+
+    def __call__(
+        self,
+        query: torch.Tensor,
+        key: torch.Tensor,
+        value: torch.Tensor,
+        out: torch.Tensor,
+        logsumexp: torch.Tensor,
+        grad_out: torch.Tensor,
+        fw_graph: Union[Callable, GraphModule],
+        joint_graph: GraphModule,
+        *other_buffers: torch.Tensor,
+    ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+        if not all(isinstance(buf, torch.Tensor) for buf in other_buffers):
+            raise RuntimeError("Other buffers must be tensors.")
+        return super().__call__(
+            query,
+            key,
+            value,
+            out,
+            logsumexp,
+            grad_out,
+            fw_graph,
+            joint_graph,
+            *other_buffers,
+        )
+
+
+templated_attention_backward = TemplatedAttentionBackwardHOP()
+templated_attention_backward.__module__ = "torch.ops.higher_order"
```

Comment:
Er, why do you need this?

Reply:
The hop? or the module rewrite thing? 
The module rewrite was copy pasta

Reply:
Found here: https://github.com/pytorch/pytorch/blob/main/torch/_higher_order_ops/while_loop.py#L63-L66
cc @ydwu4